ssv/runner.go

```diff
@@ -57,23 +54,16 @@ func NewBaseRunner(
 	controller *qbft.Controller,
 	beaconNetwork types.BeaconNetwork,
 	beaconRoleType types.BeaconRole,
-	highestDecidedSlot spec.Slot,
```

Comment:
Explanation: Since we switched to having qbft instances start with duty slot as height there is no need for highest decided anymore